### DISCUSSION TITLE
Custom variables hints are wrong in 2 locations

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#54862] [Dashboards, Panels & Visualizations]
The following 2 locations show the wrong way to create custom variables

1. Documentation
https://grafana.com/docs/grafana/latest/variables/variable-types/add-custom-variable/

```
key1 : value1,key2 : value2
```

It should be

```
key1 : value1, key2 : value2
```

Spacing is critical otherwise it won't work

2.  In the Variables creation section of a dashboard

![image](https://user-images.githubusercontent.com/900951/188927640-a782f91d-a557-4d54-b8a4-528b9f2669c7.png)

As you can see here it lists the keys first comma delimited and then the values
key1, key 2, key3 : value1, value2, value3

Thanks

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
